include/rangers-api/Application/Player/PlayerCounterTimer.h

```diff
@@ -0,0 +1,15 @@
+#pragma once
+
+namespace app::player {
+    class PlayerCounterTimer : public hh::fnd::BaseObject {
+        struct TimerValue {
```

Comment:
Starting braces should be started on a line to keep consistent with the original repository.